service/rpc2/server.go

```diff
@@ -783,6 +785,7 @@ func (s *RPCServer) ExamineMemory(arg ExamineMemoryIn, out *ExaminedMemoryOut) e
 	}
 
 	out.Mem = Mem
+	out.IsLittleEndian = isLittleEndian()
```

Comment:
Change it to `out.IsLittleEndian = true` and add this comment:

```
//TODO: get byte order from debugger.target.BinInfo().Arch
```

Reply:
done
-----------------------------------------------------------
Please let me know if I misunderstood something.